src/Pages/ContentListTable/components/EditContentModal/EditContentModal.tsx

```diff
@@ -1,326 +1,85 @@
+import { useLocation, useNavigate } from 'react-router-dom';
+import EditContentForm from './EditContentForm';
+import { EditContentRequest } from '../../../../services/Content/ContentApi';
+import { isEqual } from 'lodash';
+import { mapToContentItemsToEditContentRequest } from './helpers';
 
 const useStyles = createUseStyles({
   description: {
     paddingTop: '12px', // 4px on the title bottom padding makes this the "standard" 16 total padding
     color: global_Color_200.value,
   },
-  removeSideBorder: {
-    '&:after': {
-      borderLeft: 'none!important',
-    },
-  },
-  toggleAllRow: {
-    composes: ['$removeSideBorder'],
-    cursor: 'pointer',
-    borderBottom: 'none!important',
-    '& td': {
-      color: global_link_Color.value + '!important',
-      padding: '12px 0!important',
-    },
-    '& svg': {
-      fill: global_link_Color.value + '!important',
-      padding: '',
-    },
-  },
-  colHeader: {
-    '& td': {
-      '&:not(:last-child)': { cursor: 'pointer' },
-      padding: '12px 0!important',
-    },
-  },
-  mainContentCol: {
-    composes: ['$removeSideBorder'],
-    padding: '16px 0px 16px 36px!important',
-  },
-  toggleAction: {
-    composes: ['$removeSideBorder'],
-    '& button': {
-      padding: '8px',
-    },
-  },
   saveButton: {
     marginRight: '36px',
     transition: 'unset!important',
   },
-  singleContentCol: {
-    padding: '8px 0px 0px !important',
-  },
-  gpgKeyInput: {
-    '& .pf-c-form-control': {
-      backgroundPositionX: 'calc(100% - 1.3em)',
-    },
-  },
 });
 
-const EditContentModal = ({ values, open, setClosed }: EditContentProps) => {
-  if (!open) return <></>;
-  const { hidePackageVerification } = useAppContext();
-  const initialValues = mapToDefaultFormikValues(values);
-  const [changeVerified, setChangeVerified] = useState(false);
-  const [gpgKeyList, setGpgKeyList] = useState<Array<string>>(
-    initialValues.map(({ gpgKey }) => gpgKey),
-  );
+const EditContentModal = () => {
   const classes = useStyles();
+  const navigate = useNavigate();
   const queryClient = useQueryClient();
-  const formik = useFormik({
-    initialValues: initialValues,
-    validateOnBlur: false,
-    validateOnChange: false,
-    validationSchema: makeValidationSchema(),
-    initialTouched: values.map(() => ({ name: true, url: true })),
-    onSubmit: () => undefined,
-  });
-
-  const valuesHaveChanged = useMemo(
-    () => !isEqual(initialValues, formik.values),
-    [initialValues, formik.values],
-  );
-
-  const updateGpgKey = (index: number, value: string) => {
-    setChangeVerified(false);
-    const updatedData: Array<string> = [...gpgKeyList];
-    updatedData[index] = value;
-    setGpgKeyList(updatedData);
-  };
-  const { fetchGpgKey, isLoading: isFetchingGpgKey } = useFetchGpgKey();
-
-  const debouncedGpgKeyList = useDebounce(gpgKeyList, 300);
+  const { search } = useLocation();
+  const [initialLoad, setInitialLoad] = useState(true);
```

Comment:
I would like to clean this up, but currently the "initialLoad" prevents the modal content from being rendered (and it's child api calls) the isLoading determines if the modalContent is calling their API. If I would attempt to combine the state, it would indeed work with the initial load (that is correct), but upon any loading action on the child the modal content would unmount and turn into a spinner.